### DISCUSSION TITLE
Download v-tooltip.js?

### DISCUSSION THREAD

**Discussion by craigg75** [Akryum/v-tooltip#386] [Q&A]
Is there an unminified version of https://github.com/Akryum/v-tooltip/blob/master/dist/v-tooltip.min.js somewhere?
Thanks.

**Comment by Akryum** [comment]
https://unpkg.com/browse/v-tooltip@2.0.2/dist/

**Comment by craigg75** [comment]
That's still the minified version. Is there an full version of the minified js? Thanks.
